pkg/compute/manifest/manifest.go

```diff
@@ -62,6 +62,19 @@ func (d *Data) ServiceID() (string, Source) {
 	return "", SourceUndefined
 }
 
+// ServiceName yields a ServiceName.
+func (d *Data) ServiceName() (string, Source) {
+	if d.Flag.ServiceName != "" {
+		return d.Flag.ServiceName, SourceFlag
+	}
+
+	if d.File.ServiceName != "" {
+		return d.File.ServiceName, SourceFile
+	}
+
+	return "", SourceUndefined
+}
+
```

Comment:
As per comment below, we don't need to add or read anything from the manifest for this command. Please revert the changes to this file.